### DISCUSSION TITLE
Make the "last run" field configurable

### DISCUSSION THREAD

**Discussion by GergelyKalmar** [apache/airflow#24004] [Ideas]
**Description**

I would like to be able to configure the "Last Run" column to show the last start date/time (instead of the last execution date/time) on the home page.

**Use case / motivation**

The large majority of my scheduled pipelines transfer data from one system to another, in which case the data in the target system will be up-to-date until about the task start date. It would therefore be much more useful to me to see when was a DAG last started (the "execution time" is not really meaningful for such use cases).

Being able to switch the column so that the start date is shown by default (and the execution date/time perhaps on hover) would make the home page much more useful to us.

**Are you willing to submit a PR?**

I am not very familiar with the codebase yet, but it should be relatively straightforward for someone who knows it well.

**Related Issues**

I could not find a related issue, even though I imagine it is likely that this issue was discussed before.

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!
